src/libraries/Microsoft.Extensions.Configuration.Binder/src/ConfigurationBinder.cs

```diff
@@ -698,6 +746,37 @@ private static Array BindArray(Array source, IConfiguration config, BinderOption
                 options);
         }
 
+        // todo: steve - we might not need this; currently, these attributes are only applicable to properties and
```

Comment:
@eerhardt commented on the previous PR:

> What's the plan for addressing this? I agree that this code doesn't make sense because these attributes can't be on parameters

Part of me thinks it might be a useful addition and would bring parameter binding in-line with property binding, but another part of me thinks it might be overkill.

I'm happy to be guided on the way to go here.

Reply:
@maryamariyan do you have any thoughts here?

Reply:
I just removed this. If there's any objection, I can replace it and add appropriate tests.